data/products.json

```diff
@@ -273,5 +271,21 @@
         "link": "https://supabase.com/"
       }
     ]
+  },
+  {
+    "name": "Azure",
+    "country_code": "US",
+    "alternatives": [
+      {
+        "name": "Google Cloud",
+        "country_code": "US",
```

Comment:
We are not interested in US alternatives, and we can add other alternatives like Alibaba Cloud and OpenStack